scripts/fiss_site_tidy.R

```diff
@@ -202,22 +218,18 @@ form_fiss_site <- bind_rows(
 
 # burn to file
 # The following fields need to be added by hand in the spreadsheet:
-# UTM method, No Visible Channel, Dewatered-Dry/Int Channel, Waterbody (ID) Identifier (this should be scripted eventually)
+# UTM method, No Visible Channel, Waterbody (ID) Identifier (this should be scripted eventually)
 form_fiss_loc %>%
   readr::write_csv(paste0(
     'data/inputs_extracted/form_fiss_loc_tidy',
-    # '_',
-    # format(lubridate::now(), "%Y%m%d"),
     '.csv'),
     na = '')
 
 
 form_fiss_site %>%
   readr::write_csv(paste0(
```

Comment:
picky jerk... 

paste0 is not necessary anymore once the date stamp is removed...